src/CellGrid.java

```diff
@@ -1,2 +1,28 @@
 public class CellGrid {
+
+    private String[][] cellGrid;
+    private Cell cell;
+
+
+    public CellGrid(Cell cell){
+        int rows = 6;
+        int columns = 6;
+        this.cellGrid = new String[rows][columns];
+        this.cell = cell;
+        initialiseCellGrid();
+    }
+
+
+    public String[][] getCellGrid() {
+        return cellGrid;
+    }
+
+    private void initialiseCellGrid(){
+            for(int row = 0; row < this.cellGrid.length; row ++){
+                for(int column = 0; column < this.cellGrid[row].length; column++){
+                    cellGrid[row][column] = cell.getCell();
```

Comment:
your question around the grid and printing it comes down to this here - you're only ever working with the string representation of the cell and never the actual cell itself beyond this point here